apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -432,33 +44,104 @@ export const SemanticSlots: React.StatelessComponent<ISemanticSlotsProps> = (pro
     );
   };
 
-  const fillVariantSlotsList = (variantType: VariantThemeType, mappingType: {}): JSX.Element[] => {
+  const trimPaletteSlots = (paletteSlots: {}): {} => {
+    let trimmedPaletteSlots = {};
+    for (let palette in paletteSlots) {
+      if (
+        palette.startsWith('theme') ||
+        palette.startsWith('neutral') ||
+        palette === 'black' ||
+        palette === 'white' ||
+        palette === 'redDark' ||
+        palette === 'accent'
+      ) {
+        (trimmedPaletteSlots as any)[palette] = (paletteSlots as any)[palette];
```

Comment:
...which would let you remove the any here

Reply:
done - removed most of the anys, 4 left that I'm not sure we can avoid but if you have any ideas then I'd love to try them out! the 4 that are left are when indexing with a string key into a type like `IPalette` or `ISemanticColors`